src/setupProxy.js

```diff
@@ -0,0 +1,18 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+module.exports = function (app) {
+  app.use(
+    "/api",
+    createProxyMiddleware({
+      target: "http://localhost:8000",
+      changeOrigin: true,
+    })
+  );
+  app.use(
+    "/kvapi",
+    createProxyMiddleware({
+      target: "http://localhost:8010",
+      changeOrigin: true,
```

Comment:
NOTE: This approach will require setting up a new "network" in the dropdown with the URL of `/kvapi` before it works.